content/security/for-admins/single-sign-on/configure/configure-idp.md

```diff
@@ -94,11 +112,26 @@ See [More resources](#more-resources) for a video overview on how to set up SSO
 10. Here you can find the **SAML Sign-in URL** and the **x509 Certificate**. Open the certificate file in a text editor and paste the contents of the file in the **x509 Certificate** field in Docker Hub or Admin Console. Then, copy the value of the **SAML Sign-in URL** and paste it into the corresponding field in Docker Hub or Admin Console.
 
 {{< /tab >}}
-{{< tab name="Entra ID SAML 2.0" >}}
+{{< tab name="Entra ID SAML" >}}
+
+You can configure Docker with Entra ID using the pre-configured Docker integration, or by creating a custom application. You can use the [Docker app integration](#add-the-app-integration-in-entra-id) to simplify the set-up process.
+
+### Add the app integration in Entra ID
+
+1. Go to the Entra ID admin portal.
+2. Go to **Identity > Applications > Enterprise applications > All applications**.
+3. Select **New application**.
+4. Search for Docker and add it to the main directory.
+5. Copy the Docker SSO **Connection ID** from Docker Hub or Admin Console.
+6. Enter the SSO Connection ID in Entra ID to create your application.
+7. Once your application is created, find the **Login URL** and **SAML Signing Certificate** and copy them into the corresponding fields in your Docker SSO connection.
```

Comment:
suggestion: Specify Certificate base 64 so they don't have an issue on copy pasting the value into Docker Hub. Also change the order as they appear on Entra app
```suggestion
7. Once your application is created, find the **SAML Signing Certificate (Base 64)** and **Login URL** and copy them into the corresponding fields in your Docker SSO connection.
```